BREAKING.md

```diff
@@ -33,6 +34,17 @@ Please use `DriverErrorType.fileOverwrittenInStorage` instead of `OdspErrorType.
 ### absolutePath use in IFluidHandle is deprecated
 Rather than retrieving the absolute path, ostensibly to be stored, one should instead store the handle itself. To load, first retrieve the handle and then call `get` on it to get the actual object. Note that it is assumed that the container is responsible both for mapping an external URI to an internal object and for requesting resolved objects with any remaining tail of the external URI. For example, if a container has some map that maps `/a --> <some handle>`, then a request like `request(/a/b/c)` should flow like `request(/a/b/c) --> <some handle> --> <object> -->  request(/b/c)`.
 
+### ITelemetryProperties may be tagged for privacy purposes
+Telemetry properties on logs now may be tagged, meaning the property value may have the shape
```

Comment:
I think you really want to focus here on "breaking" part. I.e. if you do not support that new feature, then you will lose some properties in your telemetry. Here is how to support it.